src/WinRT.Runtime/ObjectReference.cs

```diff
@@ -26,6 +26,7 @@ abstract class IObjectReference : IDisposable
         private readonly IntPtr _thisPtr;
         private object _disposedLock = new object();
         private IntPtr _referenceTrackerPtr;
+        protected IntPtr _contextToken;
```

Comment:
This field is also defined in `ObjectReferenceWithContext<T>`, so it's duplicated. We need to remove one of them.

Reply:
We only need one :)